experiments/transfer_learn_benchmark.sh

```diff
@@ -67,15 +68,18 @@ done
 
 if [[ $NEED_TEST_FILES == "true" ]]; then
   # Generate quick reward models for test.
-  save_dir=tests/testdata/reward_models/${ALGORITHM}
+  # To prevent race conditions, we use a different save_dir for each process id.
+  save_dir=tests/testdata/reward_models/${ALGORITHM}/${TIMESTAMP}/$BASHPID
 
   # Wipe directories for writing later.
   if [[ -d ${save_dir} ]]; then
-    rm -r ${save_dir}
+    echo "Wiping ${save_dir}"
```

Comment:
do we still want this or was it just for debugging?

Reply:
Good catch, thanks!